sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -243,6 +259,10 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     programOffering.studyBreaks = educationProgramOffering.studyBreaks;
     programOffering.offeringDeclaration =
       educationProgramOffering.offeringDeclaration;
+    programOffering.offeringType = educationProgramOffering.offeringType;
+    programOffering.offeringStatus = educationProgramOffering.offeringStatus;
+    programOffering.assessedBy = { id: userId } as User;
```

Comment:
Every time that an offering is saved the `assessedby` will be updated?

Reply:
There was a common transformation code written. Looks like I need to update the transformation separately for create and update.